scripts/webpack/js-bundle-factory.js

```diff
@@ -91,6 +91,7 @@ class JsBundleFactory {
       },
       devtool: 'source-map',
       module: {
+        noParse: ['/*.ts/'],
```

Comment:
A few questions:

1. Why is this needed? It sounds like `noParse` is normally used to prevent Webpack from parsing large external third party libraries (https://webpack.js.org/configuration/module/#modulenoparse)
2. Is this going to break/disable source maps?
3. Doesn't this effectively negate `rules.test = /\.ts$/` below?
4. Why are there string quotes around what appears to be a RegExp literal?

Reply:
I think I had that from a little test from before. Removing it. SideNote: it doesn't seem to have any effect as it still parses the ts files...